packages/frontend/src/components/UserSettings/AccessTokensPanel/TokensTable.tsx

```diff
@@ -79,39 +88,48 @@ export const TokensTable = () => {
                     </tbody>
                 </Table>
             </Paper>
-            <Dialog
-                isOpen={!!tokenToDelete}
-                icon="delete"
+
+            <Modal
+                centered
+                opened={!!tokenToDelete}
                 onClose={() => !isDeleting && setTokenToDelete(undefined)}
-                title={`Delete token ${tokenToDelete?.description}`}
-                lazy
+                title={
+                    <Title order={4}>
+                        Delete token {tokenToDelete?.description}
+                    </Title>
+                }
             >
-                <div className={Classes.DIALOG_BODY}>
-                    <p>
-                        Are you sure ? This will permanently delete the
-                        <b> {tokenToDelete?.description} </b> token.
-                    </p>
-                </div>
-                <div className={Classes.DIALOG_FOOTER}>
-                    <div className={Classes.DIALOG_FOOTER_ACTIONS}>
+                <Stack spacing="xl">
+                    <Text>
+                        Are you sure? This will permanently delete the
+                        <Text component="b">
```

Comment:
this can be fw={600} `b` feels just too heavy

Reply:
thanks! looks good: 
![image](https://user-images.githubusercontent.com/7611706/236518669-d245fbcd-a490-4e81-8dc9-45da56089bab.png)